doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
+
+<a name="2.1"></a>
+## 2.1 Specification Types
+
+- **Reference Model (RM)**: focuses on the __**Infrastructure Abstraction**__ and how services and resources are exposed to VNFs/CNFs. It needs to be written at a high enough level that as new **Reference Architectures** and **Reference Implementations** are added, the model document should require few or no changes. Additionally, the Reference Model is intended to be neutral towards VMs or Containers.
+- **Reference Architecture (RA)**: High level Infrastructure system components and their interactions that takes the Reference Model goals and maps them to something that has components in the real world.  It is expected that at least one, but not more than a few, Reference Architecture will conform to the Reference Model.  The intention is, whenever possible, to use existing elements, rather than specify entirely new architectures in support of the high-level goals specified in the **Reference Model**.
+- **Reference Implementation(RI)**: Builds on the requirements and specifications developed in RM, RAs and adds details so that it can be implemented.  Each Reference Architecture is expected to be implemented by at least one Reference Implementation.
+- **Reference Certification(RC)**: Builds on the requirements and specifications developed in the other documents and adds details on how an implementation will be verified, tested and certified. Both infrastructure verification and certification as well as VNFs/CNFs verifications and certifications will be covered.
+
+Below is a diagram of the different artifacts that will need to be created to support the implementation of the abstract concepts presented in the **Reference Model**, which are then applied to create the **Reference Architecture**, that will be deployed using the requirements spelled out in the **Reference Implementation**.
+
+<p align="center"><img src="./figures/tech_scope_3.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 2:</b> Description of the possible different levels of CNTT artefacts</p>
+
+<a name="3.0"></a>
+## 3. Principles
+
+Any specification work created within CNTT **must** obey to set of principles specified by CNTT:
+
+* [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
+* [Reference Architecture Principles](../ref_arch#principles)
+
+<a name="4.0"></a>
+## 4. Technology Direction
+
+<a name="4.1"></a>
+### 4.1 IO Virtualization
+
+There are different ways of which IO devices (such as NICs) are presented to workloads for consumption by those workloads. Here is a list of current methods of existing IO Virtualization:
+
+- Para-Virtualization method (software only).
+- Direct assignment with Virtual Technology.
+- Device Sharing with SR-IOV & Direct Assignment.
+- Para-Virtualization method (Hardware support).
```

Comment:
Consider:
- Direct Assignment via h/w assisted PCI-Passthrough (IOMMU)
- Device Sharing with SR-IOV & h/w assisted PCI-Passthrough (IOMMU)
- Para-Virtualization method with SmartNIC and PCI-Passthrough (IOMMU)

Reply:
comments taken.